tests/excerptexport/models/test_export_model.py

```diff
@@ -0,0 +1,31 @@
+from datetime import timedelta
+
+import pytest
+
+from django.utils.datetime_safe import datetime
+
+
+@pytest.mark.django_db
+def test_created_export_has_correct_time_stamp(export):
```

Comment:
As `Export.objects.create` is the action being tested here, consider moving that from the fixture to the test body. To avoid duplication between the test and the fixture, this can be done by explicitly invoking the fixture's function. (Import it locally to avoid namespace pollution.)